js/models/ShippingAddress.js

```diff
@@ -0,0 +1,16 @@
+import BaseModel from './BaseModel';
+
+export default class extends BaseModel {
+  defaults() {
+    return {
+      Name: '',
+      Company: '',
+      AddressLineOne: '',
+      AddressLineTwo: '',
+      City: '',
+      State: '',
+      Country: '',
+      PostalCode: '',
```

Comment:
I have a bug in the server to lowercase the keyname to be consistent with other APIs.

Yeah, we could discuss v2 address fields although I think some others should be included today (maybe at least wolf and cp?).